{"schema_version":"1.3.1","id":"GO-2025-3645","modified":"2025-05-05T16:13:00Z","published":"2025-05-05T16:13:00Z","aliases":["CVE-2019-11243","GHSA-gc2p-g4fg-29vh"],"summary":"Kubernetes did not effectively clear service account credentials in k8s.io/kubernetes","details":"Kubernetes did not effectively clear service account credentials in k8s.io/kubernetes","affected":[{"package":{"name":"k8s.io/kubernetes","ecosystem":"Go"},"ranges":[{"type":"SEMVER","events":[{"introduced":"1.12.0"},{"fixed":"1.12.5"},{"introduced":"1.13.0"},{"fixed":"1.13.1"}]}],"ecosystem_specific":{}}],"references":[{"type":"ADVISORY","url":"https://github.com/advisories/GHSA-gc2p-g4fg-29vh"},{"type":"ADVISORY","url":"https://nvd.nist.gov/vuln/detail/CVE-2019-11243"},{"type":"WEB","url":"https://github.com/kubernetes/kubernetes/issues/76797"},{"type":"WEB","url":"https://security.netapp.com/advisory/ntap-20190509-0002"}],"database_specific":{"url":"https://pkg.go.dev/vuln/GO-2025-3645","review_status":"UNREVIEWED"}}